third_party/protobuf/google/protobuf/compiler/plugin.proto

```diff
@@ -0,0 +1,150 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
I've seen many projects put `protobuf proto` under `third_party`, I wonder if this is the recommended way from `protobuf` or `golang` community?

Reply:
Where does it come from? I'm thinking about when and how to update it to a newer version in the future.

Reply:
These files come from [kubernetes third party directory](https://github.com/kubernetes/kubernetes/tree/master/third_party). I was meant to put these files into `vendor`, but unfortunately the cmd `go mod vendor` does not support to add none-go files into `vendor` individually. So I have to copy them manually. I don't find any better way to handle this issue.

Reply:
What's the k/k version you are copying from? Should we also upgrade it when we update k/k dependency?

> I was meant to put these files into vendor, but unfortunately the cmd go mod vendor does not support to add none-go files into vendor individually. So I have to copy them manually. I don't find any better way to handle this issue.

Yes, we can't use `vendor`.

Reply:
k/k version 1.21. It is noticed that these files and their directory have not been changed since 6 years ago.